kratos/utilities/binbased_fast_point_locator.h

```diff
@@ -151,7 +156,7 @@ class BinBasedFastPointLocator
      * @brief Function to construct or update the search database
      * @param CellSize The current size of the cell used for search
      */
-    void UpdateSearchDatabaseAssignedSize(const double CellSize)
+    void UpdateSearchDatabaseAssignedSize(const BinsCoordinateType CellSize)
```

Comment:
FYI @RiccardoRossi. This was wrong. Seems that nobody was using it but calling it would have updated the number of cells with the size of the cell. Probably a copy paste mistake from the other locator which uses a Bins that has a constructor with `BucketSize`